.gitignore

```diff
@@ -1,3 +1,5 @@
+/node_modules/
```

Comment:
:memo: Please add a comment above this line (like in the entries below), something to the effect of:

```
# Generated by `yarn install`
```